ntcore/callback_manager.py

```diff
@@ -1,10 +1,10 @@
 # validated: 2018-11-27 DS 18c8cce6a78d cpp/CallbackManager.h
-'''----------------------------------------------------------------------------'''
-''' Copyright (c) FIRST 2017. All Rights Reserved.                             '''
-''' Open Source Software - may be modified and shared by FRC teams. The code   '''
-''' must be accompanied by the FIRST BSD license file in the root directory of '''
-''' the project.                                                               '''
-'''----------------------------------------------------------------------------'''
+"""----------------------------------------------------------------------------"""
+""" Copyright (c) FIRST 2017. All Rights Reserved.                             """
+""" Open Source Software - may be modified and shared by FRC teams. The code   """
+""" must be accompanied by the FIRST BSD license file in the root directory of """
+""" the project.                                                               """
+"""----------------------------------------------------------------------------"""
```

Comment:
Hmm. Shouldn't this license block be a comment block rather than strings?

Reply:
It was previously a string, so this PR doesn't change that.

Reply:
Fair enough. We should probably fix this though, as this doesn't seem like an appropriate docstring.